src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -3108,7 +3111,8 @@ void ShenandoahHeap::rebuild_free_set(bool concurrent) {
   size_t region_size_bytes = ShenandoahHeapRegion::region_size_bytes();
   ShenandoahHeapLocker locker(lock());
   size_t young_cset_regions, old_cset_regions;
-  _free_set->prepare_to_rebuild(young_cset_regions, old_cset_regions);
+  size_t first_old_region, last_old_region, old_region_count;
+  _free_set->prepare_to_rebuild(young_cset_regions, old_cset_regions, first_old_region, last_old_region, old_region_count);
```

Comment:
Since we don't really need the first_old_region() and last_old_region() methods of ShenandoahFreeSet, they are removed, so we cannot write this assertion.  I'll add a "comparable" assertion to assure sanity.